ssl/statem/statem_lib.c

```diff
@@ -704,6 +704,19 @@ static int version_cmp(const SSL *s, int a, int b)
     return DTLS_VERSION_LT(a, b) ? -1 : 1;
 }
 
+int ssl_strict_version_check(const SSL *s, int version)
```

Comment:
May I ask you to elaborate on this?

It really looks like that `min`/`max` versions will be handled later in `ssl_choose_client_version` during `tls_process_server_hello`. Is there any point in doing them during `ssl_strict_version_check`?